### PR TITLE
Experience section mobile

### PR DESCRIPTION
**Pull request by 3Z3QU13L5**
Mobile Version of the experience section, I eliminate the professional, since it was a little distracting when I was doing the modifications, 🤔 but now I remember that I could just change the root/origin of the branch 😥

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -223,7 +223,7 @@ <h3>
               <li class="experince__content--right">
                 <div>
                   <div class="experince__point"></div>
-                  <div class="experince__text">
+                  <div class="experince__text experince__right--text">
```
Comment (fxisco) [inline Index.html]: I'm little confused is right a specific element? or a modifier? if is a modifier it should be `experience__text--right`

File: styleSheet.css [hunk 2]
```diff
@@ -258,36 +252,18 @@ ul {
...
 #experince {
   background-image: url('images/experinceSection_backgroudImage.jpg');
   background-position: center top;
   background-repeat: no-repeat;
   background-size: cover;
-  height: 200vh;
+  height: calc(60em + 40vh);
+}
+
+.experince__backgroundOpacity {
```
Comment (fxisco) [inline styleSheet.css]: I think opacity could be a modifier or maybe you could leave it with just `.experince__background`, this way if you decide to change the style in the future you will not have to change the name

File: Index.html (hunk 1 above)
Comment (3Z3QU13L5) [reply Index.html]: ha aha aha, I forget to eliminate that class from the element, I found a different way without using that class.

File: styleSheet.css (hunk 2 above)
Comment (3Z3QU13L5) [reply styleSheet.css]: I think I will leave it as `.experince__background`.

**Pull request merged**